perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -454,6 +454,23 @@ sub inspect : Private {
                 $redirect_uri = $c->uri_for_action('my/planned');
             }
 
+            # Sign up to alerts by default
+            my $options = {
```

Comment:
I hear ya. I'm very much in favour of taking as much logic out of the controller as possible, so I'll take a look at this 👍